jsk_arc2017_baxter/euslisp/lib/pick-interface.l

```diff
@@ -130,6 +130,7 @@
     ;; TODO: in-hand verification
     ;; TODO:   target-obj = in-hand object  -> :set_target_cardboard
     ;; TODO:   target-obj != in-hand object -> :return_object
+    (setq target-obj t)
```

Comment:
The return value of `:verify-object` is used for state machine.
If we set `target-obj` as object name, state machine dies.
In stow-interface, you set `t`:
https://github.com/start-jsk/jsk_apc/blob/master/jsk_arc2017_baxter/euslisp/lib/stow-interface.l#L104